pkgs/development/python-modules/weasyprint/default.nix

```
@@ -0,0 +1,48 @@
+{ stdenv, fetchPypi, buildPythonPackage
+, pytestrunner
+, cffi, pyphen, cairosvg, html5lib, tinycss2, cairocffi, cssselect2
+, pytestcov, pytest-flake8, pytest-isort }:
+
+# python3 only
+buildPythonPackage rec {
+  pname = "weasyprint";
+  version = "44";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "WeasyPrint";
+    sha256 = "0f5bzwac1c4wgsscdv6d2d08n6g3j7pmj76ja5q31v40vzkz96vr";
+  };
+
+  buildInputs = [
+    pytestrunner
+  ];
+
+  propagatedBuildInputs = [
+    cffi
+    pyphen
+    cairosvg
+    html5lib
+    tinycss2
+    cairocffi
+    cssselect2
+  ];
+
+  # OSError: cannot load library 'libgobject-2.0.so.0': libgobject-2.0.so.0: cannot open shared object file: No such file or directory
+  # DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
+  # from collections import Mapping
+  doCheck = false;
+  checkInputs = [
+    pytestcov
+    pytestrunner
+    pytest-flake8
+    pytest-isort
+  ];
+
+  meta = with stdenv.lib; {
```

```suggestion
  meta = with lib; {
```
Please also do so for the other packages.